content/shmem_ctx.tex

```diff
@@ -0,0 +1,133 @@
+\color{ForestGreen}
+
+All \openshmem RMA, AMO, and memory ordering routines are
+performed on a communication context.  The communication context defines an
+independent ordering and completion environment, allowing users to manage the
+overlap of communication with computation and also to manage communication
+operations performed by separate threads within a multithreaded \ac{PE}.  For
+example, in single-threaded environments, contexts can be used to pipeline
+communication and computation.  In multithreaded environments, contexts can
+additionally provide thread isolation, eliminating overheads resulting from
+thread interference within the communication subsystem.
+
+Context handles are of type \CTYPE{shmem\_ctx\_t} and are valid for
+language-level assignment and comparison.  A handle to the desired context is
+passed as an argument in the \Cstd \CTYPE{shmem\_ctx\_*} and type-generic API
+routines.  API routines that do not accept a context argument operate on the
+default context.  The default context can be used explicitly through the
+\const{SHMEM\_CTX\_DEFAULT} handle.
```

Comment:
`\const` --> `\CONST`
We should DocEdit this command since the styling for `\const` is better but `\CONST` is already used everywhere except the Library Constants section.